yaml2rst/rst_writer.py

```diff
@@ -87,11 +87,13 @@
     'False': 'No',
 }
 
+
 TO_COLLAPSE = ['items']
 
 
 _REF = ':ref:'
-ref_pattern = re.compile("{}`\S+`".format(_REF))
+# does not match absolute ref link e.g. :ref:'filename#/path/to/prop'
+ref_pattern = re.compile(":ref:`[a-zA-Z0-9/_-]+`".format(_REF))
```

Comment:
This is wrong:
* you removed the `{}` in the format string but left the `format`.
* The regexp is wrong. What distinguishes a relative from an absolute ref is whether it contains a `#` before the first slash. The best way would probably be to have a general pattern here, like 
```
r":ref:`([^`])*?`
```
Note the `r` in front to signal a raw string, to avoid having to double escape backslashes (a good habit to take for regexes), and the `*?` which is a non-greedy variant of `*`. The `()` are not needed but might be useful if you want to use a the contents through a capture group (with `ref_pattern.group(1)`).
Later, you can use another pattern, like
```
r'[^#/]*#.*'
```
on the captured string which matches when the link is absolute.